cypress/e2e/models/administration/custom-migration-targets/custom-migration-target.ts

```
@@ -0,0 +1,249 @@
+
+export interface CustomMigrationTarget {
+    name: string;
+    description?: string;
+    imagePath?: string;
+    ruleTypeData: RulesRepositoryFields | RulesManualFields;
+    language: Languages;
+    sources?: string[];
+    targets?: string[];
+}
+
+export class CustomMigrationTarget {
```

Biome (2.1.2)

[error] This class is unsafely merged with an interface.

The interface is declared here.

The TypeScript compiler doesn't check whether properties defined in the interface are initialized in the class.

(lint/suspicious/noUnsafeDeclarationMerging)